kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/SinkPropertiesFactory.java

```diff
@@ -59,14 +59,9 @@ public void testProperties(BigQuerySinkConfig config) {
 
     config.getMap(config.DATASETS_CONFIG);
```

Comment:
shouldn't we test if the value in the config is equal to the default here?

Reply:
This method is only used in [one test](https://github.com/confluentinc/kafka-connect-bigquery/blob/986995e2b8eb35b73a712d959a70b0f0e7f0e8fa/kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfigTest.java#L46) whose purpose is just to ensure that we can invoke these methods without any errors; verifying the semantics of these methods (and any downstream logic) is left to other tests.